pkg/plugins/resources/temurin/spec.go

```diff
@@ -0,0 +1,94 @@
+package temurin
+
+type Spec struct {
+	// ReleaseLine specifies the line of Temurin release to retrieve.
+	//
+	// default: "lts"
+	//
+	// Allowed values:
+	// * "lts"
+	// * "feature"
+	ReleaseLine string `yaml:",omitempty"`
+	// ReleaseType specifies the type of Temurin release to retrieve.
+	//
+	// default: "ga"
+	//
+	// Allowed values:
+	// * "ga" (General Availability)
+	// * "ea" (Early Availability, e.g. nightly builds)
+	ReleaseType string `yaml:",omitempty"`
```

Comment:
```suggestion
	ReleaseType string `yaml:",omitempty"  jsonschema:"required"`
```

If this field is required then this tag will trigger a warning if not set

Reply:
Oh thanks! Let me set up required/non required attributes!

Reply:
Nevermind: field is not mandatory as we can have the following resource (with no spec at all):

```yaml
sources:
  ## Returns last LTS (implicit default)
  getLastDefaultVersion:
    kind: temurin
```

=> so no attribute mandatory